Cargo.toml

```diff
@@ -43,7 +43,7 @@ dpdk-sysroot-helper = { path = "./dpdk-sysroot-helper", package = "dataplane-dpd
 dplane-rpc = { git = "https://github.com/githedgehog/dplane-rpc.git", version = "1.1.2" }
 errno = { path = "./errno", package = "dataplane-errno" }
 flow-info = { path = "./flow-info", package = "dataplane-flow-info" }
-gateway_config = { git = "https://github.com/githedgehog/gateway-proto", tag = "v0.14.0", version = "0.14.0"}
+gateway_config = { git = "https://github.com/githedgehog/gateway-proto", tag = "v0.14.0" }
```

Comment:
The `Cargo.lock` changes in this commit seem unrelated to this change. Please leave them out, or move them to a separate dependency-bump commit.

Reply:
I just reran the lockfile generator to confirm and the changes appear exactly as they show in this commit :shrug:

Reply:
OK my bad, sorry for the noise :face_with_peeking_eye: